README-ruRU.md

```diff
@@ -3683,9 +3682,9 @@
 
   linux_organization = Organization.find(...)
 
-  # плохо - нарушает private области видимости
+  # плохо - нарушает приватные области видимости
   linux_organization.send(:reset_token)
-  # хорошо - выдаст exception
+  # хорошо - выдаст исключение
```

Comment:
Please change the notes to the parens style, we try to write `bad (some description)` in the whole guide. We still have a lot of inconsistencies, but let's try to keep this style.